src/server/api/controllers/vessel.controller.js

```diff
@@ -68,14 +68,12 @@ const editVessel = async ({ body }) => {
   const vessel = await knex
     .from("vessels")
     .select("*")
```

Comment:
You can delete `.select(*)` as well, because in knex you don't need it.

You can take a look at this link: [Knex cheatsheet](https://devhints.io/knex)

Reply:
Ok ... Thanks